gspread/auth.py

```diff
@@ -162,8 +163,12 @@ def oauth(
 
             * `%APPDATA%\gspread\authorized_user.json` on Windows
             * `~/.config/gspread/authorized_user.json` everywhere else
+    :type client_factory: :class:`gspread.ClientFactory`
+    :param client_factory: A factory function that returns a client class.
+        Defaults to :class:`gspread.Client` (but could also use
+        :class:`gspread.BackoffClient` to avoid rate limiting)
 
-    :rtype: :class:`gspread.Client`
+    :rtype: :class:`gspread.ClientFactory`
```

Comment:
same here it should be: `` :class:`gspread.client.Client` ``

Reply:
Done.